nilearn/decoding/decoder.py

```diff
@@ -330,6 +330,18 @@ def __init__(self, estimator='svc', mask=None, cv=10, param_grid=None,
         self.n_jobs = n_jobs
         self.verbose = verbose
 
+    def _check_estimator(self):
+        if not isinstance(self.estimator, _basestring):
+            warnings.warn('Use a custom estimator at your own risk.')
+
+        elif self.estimator in list(SUPPORTED_ESTIMATORS.keys()):
+            estimator = SUPPORTED_ESTIMATORS.get(self.estimator,
+                                                 self.estimator)
+        else:
+            raise ValueError("The list of known estimator is: %s" %
+                             list(SUPPORTED_ESTIMATORS.keys()))
```

Comment:
Unnecessary `list` typecast.

Reply:
This one is used because when printing the `some_dictionary.keys()` it will return the `dict_keys([...])`. So adding a `list` typecast will make the output cleaner by removing the `dict_keys` part.

Reply:
RIght. Thanks!